tests/conftest.py

```diff
@@ -72,11 +74,44 @@ def compile_and_run(device, reset_torch_dynamo, request):
             accuracy = calculate_accuracy(outputs, outputs_after)
             if accuracy:
                 comp_runtime_metrics["accuracy"] = accuracy
+
+            # Memory analysis
+            mm = option._memory_manager
+            compiled_memory_metric = {}
+            peak_usage = mm.peak_sram_usage / 1048576
+            peak_sram_usage_metric = {"peak_sram_usage": peak_usage}
+
+            if mem_utils.check_sram_overflow(mm) is True:
+                compiled_memory_metric = {"fits_in_memory": "No"}
+            else:
+                compiled_memory_metric = {"fits_in_memory": "Yes"}
+
+            # These are for plotting charts for later inspection
+            from tools.plot_chart import plot_bar_chart, plot_line_chart
+
+            bar_chart_file = f"metrics/{model_name}/bar_chart.png"
+            line_chart_file = f"metrics/{model_name}/line_chart.png"
```

Comment:
can you chart if svg please?

Reply:
I tried to chart SVG. But the file size came out to be even more than PNG.